### PR TITLE
[CUMULUS-1916] Update reconciliation report columns

### PR DESCRIPTION
**Pull request by dopeters**
This adds additional columns to the list reconciliation reports page. This requires the new API to display the correct information.

Also some slight tweaks to the table sorting.

### CODE REVIEW

**Comment by jjmccoy** [comment]
UI/UX: LTG at this stage

File: app/src/js/reducers/reconciliation-reports.js [hunk 1]
```diff
@@ -45,10 +45,7 @@ export default createReducer(initialState, {
     };
   },
   [RECONCILIATIONS]: (state, action) => {
-    // response.results is a array of string filenames
-    const reports = action.data.results.map((filename) => ({
-      reconciliationReportName: filename,
-    }));
+    const reports = action.data.results;
```
Comment (flamingbear) [inline app/src/js/reducers/reconciliation-reports.js:48]: Can you write a simple reducer test for this one?

**Comment by dopeters** [comment]
@jennyhliu Thanks! Created a new `dateOnly` function to format the dates. The `Download Report` and `Delete Report` columns will be add in [CUMULUS-1917](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1917) and [CUMULUS-1918](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1918) respectively.
> Tested against my cumulus API endpoint. Notice that:
> 
> 1. `Date Generated` column don't have the correct format
> 2. `Download Report` and `Delete Report` columns are empty.
> 
> Screen Shot 2020-05-21 at 12 17 40 PM

File: app/src/js/reducers/reconciliation-reports.js (hunk 1 above)
Comment (dopeters) [reply app/src/js/reducers/reconciliation-reports.js:48]: Done

**Comment by jennyhliu** [comment]
> @jennyhliu Thanks! Created a new `dateOnly` function to format the dates. The `Download Report` and `Delete Report` columns will be add in [CUMULUS-1917](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1917) and [CUMULUS-1918](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1918) respectively.
> 
> > Tested against my cumulus API endpoint. Notice that:
> > 
> > 1. `Date Generated` column don't have the correct format
> > 2. `Download Report` and `Delete Report` columns are empty.
> > 
> > Screen Shot 2020-05-21 at 12 17 40 PM

The date format is now 05/21/20, and it should be 05/21/2020 base on the design

**Comment by dopeters** [comment]
> > @jennyhliu Thanks! Created a new `dateOnly` function to format the dates. The `Download Report` and `Delete Report` columns will be add in [CUMULUS-1917](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1917) and [CUMULUS-1918](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1918) respectively.
> > > Tested against my cumulus API endpoint. Notice that:
> > > 
> > > 1. `Date Generated` column don't have the correct format
> > > 2. `Download Report` and `Delete Report` columns are empty.
> > > 
> > > Screen Shot 2020-05-21 at 12 17 40 PM
> 
> The date format is now 05/21/20, and it should be 05/21/2020 based on the design

Ahhh good call, updated it to use the full year

**Comment by jennyhliu** [comment]
> > > @jennyhliu Thanks! Created a new `dateOnly` function to format the dates. The `Download Report` and `Delete Report` columns will be add in [CUMULUS-1917](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1917) and [CUMULUS-1918](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1918) respectively.
> > > > Tested against my cumulus API endpoint. Notice that:
> > > > 
> > > > 1. `Date Generated` column don't have the correct format
> > > > 2. `Download Report` and `Delete Report` columns are empty.
> > > > 
> > > > Screen Shot 2020-05-21 at 12 17 40 PM
> > 
> > 
> > The date format is now 05/21/20, and it should be 05/21/2020 based on the design
> 
> Ahhh good call, updated it to use the full year

Something is wrong.  The reports listed now are the same except the name.
Screen Shot 2020-05-21 at 2 39 33 PM

**Comment by jennyhliu** [comment]
> > > > @jennyhliu Thanks! Created a new `dateOnly` function to format the dates. The `Download Report` and `Delete Report` columns will be add in [CUMULUS-1917](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1917) and [CUMULUS-1918](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1918) respectively.
> > > > > Tested against my cumulus API endpoint. Notice that:
> > > > > 
> > > > > 1. `Date Generated` column don't have the correct format
> > > > > 2. `Download Report` and `Delete Report` columns are empty.
> > > > > 
> > > > > Screen Shot 2020-05-21 at 12 17 40 PM
> > > 
> > > 
> > > The date format is now 05/21/20, and it should be 05/21/2020 based on the design
> > 
> > 
> > Ahhh good call, updated it to use the full year
> 
> Something is wrong. The reports listed now are the same except the name.
> Screen Shot 2020-05-21 at 2 39 33 PM

Actually, if I remove the time filter, the reports have different type and status, but the `Date Generated` is the same.

**Comment by dopeters** [comment]
> > > > > @jennyhliu Thanks! Created a new `dateOnly` function to format the dates. The `Download Report` and `Delete Report` columns will be add in [CUMULUS-1917](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1917) and [CUMULUS-1918](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1918) respectively.
> > > > > > Tested against my cumulus API endpoint. Notice that:
> > > > > > 
> > > > > > 1. `Date Generated` column don't have the correct format
> > > > > > 2. `Download Report` and `Delete Report` columns are empty.
> > > > > > 
> > > > > > Screen Shot 2020-05-21 at 12 17 40 PM
> > > > 
> > > > 
> > > > The date format is now 05/21/20, and it should be 05/21/2020 based on the design
> > > 
> > > 
> > > Ahhh good call, updated it to use the full year
> > 
> > 
> > Something is wrong. The reports listed now are the same except the name.
> > Screen Shot 2020-05-21 at 2 39 33 PM
> 
> Actually, if I remove the time filter, the reports have different type and status, but the `Date Generated` is the same.

Thank you for this. I was referencing a different version of accessor and thought I could just pass in the function, but it needs to explicitly call the function on `row.createdAt`. Sorry about that! Should be fixed but let me know if you notice anything else.

**Comment by jennyhliu** [comment]
LGTM

**Pull request merged**